api/spec/api_docs/definitions/resource.rb

```diff
@@ -68,7 +71,12 @@ def write_only_attributes
         const_defined?(:WRITE_ONLY) ? self::WRITE_ONLY : []
       end
 
+      def writable_attributes
+        const_defined?(:WRITEABLE) ? self::WRITEABLE : []
```

Comment:
Is `WRITABLE` a good name and clear enough description of what these attributes should be?

Reply:
I don't understand why we need WRITABLE at all. Didn't I see a mechanism for setting attributes to READ_ONLY in a previous commit? Explain why we need READ ONLY and WRITABLE. We should only need one or the other, but not both.

Reply:
In the case of the comments at least, I had to write out all attributes except for the 3 in the `READ_ONLY` section.
With `WRITABLE`, I can just get the attributes that the validation accepts. Because there are less writable attributes than read only, it makes it easier to see what is there rather than what is not.

I was thinking for some serializer resources, it will make more sense to use `READ_ONLY` and for others `WRITABLE` depending on the ratio between the two.